src/session/types.ts

```diff
@@ -0,0 +1,29 @@
+// arbitrary map type of string => T
+export interface IMap<T> {
+    [key: string]: T;
+}
+
+export interface IAccessToken {
+    token?: string;
```

Comment:
Do the question marks actually do anything here? I thought they only come into play/become relevant when accessing sub-attributes or a potentially null object.